projects/plugins/jetpack/_inc/lib/markdown/extra.php

```diff
@@ -1926,7 +1926,7 @@ function _hashHTMLBlocks_inMarkdown($text, $indent = 0,
 			$parsed .= $parts[0]; # Text before current tag.
 
 			# If end of $text has been reached. Stop loop.
-			if (count($parts) < 3) {
+			if ((is_countable($parts) ? count($parts) : 0) < 3) {
```

Comment:
Stupid code transform. This would make a lot more sense.
```suggestion
			if ( ! is_countable($parts) || count($parts) < 3 ) {
```
OTOH, again we've already used `$parts` as an array just above though, so it's kind of late to test now.